framework/src/source/BaseTestSuite.bs

```diff
@@ -267,7 +267,7 @@ namespace rooibos
       if m.catchCrashes and not test.noCatch and not m.noCatch
         try
           test.run()
-          if m.isAutoAssertingMocks = true
+          if m.isAutoAssertingMocks = true and not test.isAsync
```

Comment:
With the current set up mocks and stubs will be cleaned the moment after calling the async test. Meaning this are nearly guaranteed to run into errors or failures if the async test is relying on them. For not I opted to disable the auto mock logic here and plan to make some updates/fixes to async tests overall. Like running this after the async test has finished or failed.